src/pages/get.astro

```diff
@@ -1,68 +1,184 @@
 ---
-import '../global.css'
-import '../legacy/get.scss'
-import { getSession } from '../lib/game-saving/account'
-import { getConsolesRemaining } from '../lib/remaining-consoles'
-import Navbar from '../components/navbar-main'
-import StandardHead from '../components/standard-head.astro'
-
-const remainingConsoles = await getConsolesRemaining()
-const session = await getSession(Astro.cookies)
+import "../global.css";
+import "../legacy/get.scss";
+import { getSession } from "../lib/game-saving/account";
+import { getConsolesRemaining } from "../lib/remaining-consoles";
+import Navbar from "../components/navbar-main";
+import StandardHead from "../components/standard-head.astro";
+
+const remainingConsoles = await getConsolesRemaining();
+const session = await getSession(Astro.cookies);
 ---
 
-<html lang='en'>
+<html lang="en">
 	<head>
-		<StandardHead title='Get a Sprig' />
+		<StandardHead title="Get a Sprig" />
 	</head>
 	<body>
 		<Navbar transparent session={session} />
-		
+
 		<div class="wrapper">
 			<div class="heading-outer">
 				<div class="heading-inner">
 					<h1>Share your game, get a Sprig</h1>
-					<p>After you create your own game, we'd love for you to share it with other makers on the internet!</p>
-		
 					<p>
-						If you're looking to submit your game to get a Sprig, make sure to read over the <a
-							href="https://github.com/hackclub/sprig/blob/main/docs/GET_A_SPRIG.md" target="_blank">rules here</a
+						After you create your own game, we'd love for you to
+						share it with other makers on the internet!
+					</p>
+
+					<p>
+						If you're looking to submit your game to get a Sprig,
+						make sure to read over the <a
+							href="https://github.com/hackclub/sprig/blob/main/docs/GET_A_SPRIG.md"
+							target="_blank">rules here</a
 						>.
 					</p>
 
 					<p>
-						<strong>{remainingConsoles}</strong> teenagers have already claimed their sprig!
+						<strong>{remainingConsoles}</strong> teenagers have already
+						claimed their sprig!
 					</p>
 				</div>
 			</div>
-		
+
 			<div class="inner">
 				<div class="step">
 					<div>
 						<p class="step-number">Step 0:</p>
-						<h2>Download your game file</h2>
-						<p>In the top left Sprig menu, press "Download". Your file should start downloading immediately.</p>
+						<h2>Fill in the game metadata</h2>
+						<p>
+							In the header of your game you should see a section
+							about game metadata. In order to publish your game
+							to the sprig gallery website you will need to fill
+							it in according to the following instructions. For
+							any assistance feel free to sent a message on the
+							<a href="https://hackclub.com/slack/" target="_blank">#sprig</a> channel on the <a href="https://hackclub.com/slack/" target="_blank">slack</a>
```

Comment:
replace the link for `#sprig` with `https://hackclub.slack.com/archives/C02UN35M7LG`